src/server/encode.rs

```diff
@@ -15,7 +15,7 @@ use crate::date::fmt_http_date;
 ///
 /// This is returned from [`encode`].
```

Comment:
This comment also needs to be adjusted since currently it links to nothing in the docs.

![Screenshot_2020-06-30 async_h1 server Encoder - Rust](https://user-images.githubusercontent.com/2467194/86095709-9ed62200-bab2-11ea-8746-cda46f6f5ea7.png)

Reply:
I have removed the bad part of this doc comment for now - this can always be better documented down the line